package-builder/flox-build.mk

```
@@ -377,6 +394,76 @@ define BUILD_nix_sandbox_template =
 
 endef
 
+# The following template renders targets for the sandbox build mode.
+define BUILD_nix_expression_template =
+
+  $(eval _pvarname = $(subst -,_,$(_pname)))
+
+ 	$(eval _result = result-$(_pname))
+  $(eval $(_pvarname)_tmpBasename = $(TMPDIR)/$($(_pvarname)_shortHash)-$(_pname))
+  $(eval $(_pvarname)_buildMetaJSON = $($(_pvarname)_tmpBasename)-build-meta.json)
+
+  .PHONY: $(_pvarname)_nix_sandbox_build
+  $(_pvarname)_nix_expression_build: $($(_pvarname)_buildScript) $($(_pvarname)_src_tar) \
+		$(if $(_do_buildCache),$($(_pvarname)_buildCache))
+	@echo "Building $(_pname) in Nix expression (pure) mode"
```

It's not wrong to say that it's pure, but it's redundant as all Nix expression builds are pure by definition. Just wanting to avoid confusing NEF builds with either of the pure or impure (or eventually, the enforce or warn) sandbox modes.
```suggestion
	@echo "Building $(_pname) in Nix expression mode"
```

---

Well at this moment it's also wrong.. since the nix expressions are essentially evaluated with an implicit --impure, so yeah.

Again, verbiage is a result of c/p ing existing pieces